scripts/run-with-strimzi.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+#
+# Copyright Kroxylicious Authors.
+#
+# Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+#
+
+set -eo pipefail
+
+if [[ -z $QUAY_ORG ]]; then
+  echo "Please set QUAY_ORG, exiting"
+  exit 1
+fi
+
+SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+. "${SCRIPT_DIR}/common.sh"
+cd "${SCRIPT_DIR}/.."
+
+if [[ "${QUAY_ORG}" != 'kroxylicious' ]]; then
+  echo "building and pushing image to quay.io"
+  PUSH_IMAGE=y "${SCRIPT_DIR}/deploy-image.sh"
+else
+  echo "QUAY_ORG is kroxylicious, not building/deploying image"
+fi
+${MINIKUBE} start --memory=4096
```

Comment:
I'm not sure we should be hard coding the memory here, as it overrides what ever user configuration is supplied.

If we wanted we could do something like `minikube config get memory` and make sure it is at leas 4096?

Reply:
Done, I've made it prefer the user configuration if its >= 4096